src/content/reference/manifest.mdx

```diff
@@ -509,6 +509,31 @@ metadata:
     custom.label/dev: "true"
 ```
 
+## mode (string, optional)
```

Comment:
```suggestion
#### mode (string, optional)
```
needs to be formatted under the dev section instead of in the root of the content tree